backend/app/controllers/campaigns_controller.rb

```diff
@@ -1,15 +1,12 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
 class CampaignsController < ApplicationController
```

Comment:
For all methods here, we should avoid using `params`. Let's use `campaign_params` so that unwanted params are caught.